index.js

```diff
@@ -799,20 +769,19 @@
 
   //# compose :: (b -> c) -> (a -> b) -> a -> c
   //.
-  //. Takes two functions assumed to be unary and a value of any type,
-  //. and returns the result of applying the first function to the result
-  //. of applying the second function to the given value.
-  //.
-  //. In general terms, `compose` performs right-to-left composition of two
-  //. unary functions.
+  //. Composes two unary functions, from right to left. Equivalent to Haskell's
+  //. `(.)` function.
   //.
-  //. See also [`B`](#B) and [`pipe`](#pipe).
+  //. This is the B combinator from combinatory logic.
```

Comment:
Let's keep the link to `pipe`:

_See also [`pipe`][1]._


[1]: https://sanctuary.js.org/#pipe